op-challenger/config/config.go

```diff
@@ -4,6 +4,7 @@ import (
 	"errors"
 	"fmt"
 
+	"github.com/ethereum-optimism/optimism/op-node/chaincfg"
```

Comment:
It feels weird to add the op-node as a dependency here, but I see the tradeoff between added dependency and duplicated code. It also makes sense that a challenge agent should only play on the networks available with derivation clients.

Reply:
Yeah good point. I suspect chain configs should move to their own module in op-service.  There's a weird split in op-program too where it uses the rollup config from op-node and has its own list of l2 chain configs. Would make more sense to have them both together.

Not going to try and tackle that in this PR but will have a play with it in a follow up PR and see if it works.